kinode.wit

```diff
@@ -106,40 +108,49 @@ interface standard {
         no-file-at-path,
     }
 
-    //
-    // System utils:
-    //
+    /*      ˗ˏˋ ♡ ˎˊ˗     */
+    /*    System Utils    */
+    /*      ˗ˏˋ ♡ ˎˊ˗     */
 
+    // Prints to the terminal at a given verbosity level.
+    // Higher verbosity levels print more information.
+    // Level 0 is always printed -- use sparingly.
     print-to-terminal: func(verbosity: u8, message: string);
 
-    //
-    // Process management:
-    //
+    // Returns the address of the process.
+    our: func() -> address;
 
-    set-on-exit: func(on-exit: on-exit);
+    /*         ˗ˏˋ ♡ ˎˊ˗        */
+    /*    Process Management    */
+    /*         ˗ˏˋ ♡ ˎˊ˗        */
 
     get-on-exit: func() -> on-exit;
 
+    set-on-exit: func(on-exit: on-exit);
+
     get-state: func() -> option<list<u8>>;
 
     set-state: func(bytes: list<u8>);
 
     clear-state: func();
 
     spawn: func(
+        // name is optional. if not provided, name will be a random u64.
```

Comment:
Should these "in-line" comments also follow the capitalization convention we are using in the "top-level" comments?

Reply:
I differentiated on purpose